test/model/store_test.dart

```diff
@@ -757,11 +781,7 @@ void main() {
     }
 
     void prepareExpiredEventQueue() {
-      connection.prepare(httpStatus: 400, json: {
-        'result': 'error', 'code': 'BAD_EVENT_QUEUE_ID',
-        'queue_id': updateMachine.queueId,
-        'msg': 'Bad event queue ID: ${updateMachine.queueId}',
-      });
+      connection.prepare(apiException: eg.apiExceptionBadEventQueueId());
```

Comment:
nit: Seems like this is more realistic when it uses the update machine's actual queue ID.